Example/Authenticator/Utils/Helpers/ReachabilityManager.swift

```diff
@@ -68,3 +68,7 @@ class ReachabilityManager {
         NotificationsHelper.removeObserver(self)
     }
 }
+
+protocol ReachabilityManagerProtocol {
```

Comment:
Move protocol on top of the class. Also rename it to `Connectable`. And rename `isReachable` to `isConnected`.

Reply:
I specifically made the variable as in the manager itself so that it was immediately clear what it refers to.

Reply:
and maybe it would be better to call `ConnectivityProtocol`?

Reply:
There are specific guidelines how to name protocols in swift. For example we don't add postfix "protocol" in protocols.

https://softwareengineering.stackexchange.com/questions/263521/swift-protocol-naming-conventions/263525

And after manager will be renamed to `ConnectivityManager`, variable `isReachable` should be renamed to `isConnected`.